tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -850,6 +851,39 @@ func printActionDeleted(entityName string) {
             }))
 }
 
+// Check if the specified action is a web-action
+func IsWebAction(client *whisk.Client, qname *QualifiedName) bool {
+    var isWebAction = false;
+
+    savedNs := client.Namespace
+    client.Namespace = qname.namespace
+    fullActionName := "/" + qname.namespace + "/" + qname.entityName
+
+    action, _, err := client.Actions.Get(qname.entityName)
+    if err != nil {
+        whisk.Debug(whisk.DbgError, "client.Actions.Get(%s) error: %s\n", fullActionName, err)
+        whisk.Debug(whisk.DbgError, "Unable to obtain action '%s' for validation\n", fullActionName)
+    } else {
+        weVal := getValue(action.Annotations, "web-export")
+        if (reflect.TypeOf(weVal) == nil) {
```

Comment:
@dubeejw The getValue() can return an empty interface if the annotation does not exist.  I found that comparing this directly to `nil` is not possible due to a type mismatch.  Reflection seemed to be the best solution.  Did you have something else in mind?
http://stackoverflow.com/questions/13901819/quick-way-to-detect-empty-values-via-reflection-in-go